docs/guides/integrating-cert-manager.md

```diff
@@ -0,0 +1,290 @@
+of the most important steps in this process is implementing HTTPS (HTTP over TLS/SSL) for all communication between your
+server and clients. This encrypts the data transmitted between the two, preventing eavesdropping and tampering. To do
+this, you need an SSL/TLS certificate from a trusted Certificate Authority (CA). However, issuing and managing
+certificates can be a complicated manual process. Luckily, there are many services and tools available to simplify and
+automate certificate issuance and management.
+
+This guide will demonstrate how to configure HTTPS for your application using Gateway resources managed by NGINX
+Kubernetes Gateway (NKG); [Let’s Encrypt](https://letsencrypt.org) as the Certificate Authority (CA) issuing the
+certificates used in securing these resources; and [cert-manager](https://cert-manager.io) to automate the provisioning
+and management of these certificates.
+
+## Prerequisities
+
+1.	Administrator access to a Kubernetes cluster is required.
+2.	[Helm](https://helm.sh) and [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) must be installed locally.
+3.	Deploy NGINX Kubernetes Gateway (NKG) following the [deployment instructions](../installation.md).
+4.	A DNS resolvable domain name is required. It must resolve to the public endpoint of the NKG deployment, and this
+    public endpoint must be an external IP address or alias accessible over the internet. The process here will depend
+    on your DNS provider. This DNS name will need to be resolvable from the Let’s Encrypt servers, which may require
+    that you wait for the record to propagate before it will work.
+
+## Overview
+
+![cert-manager ACME Challenge and certificate managment with Gateway API](/docs/images/cert-manager-gateway-workflow.png)
+
+At a high level, the process looks like this:
+
+1. We deploy cert-manager and create a ClusterIssuer which specifies Let’s Encrypt as our CA and Gateway as our ACME
+   HTTP01 Challenge solver
+2. We create a Gateway resource for our domain (cafe.example.com) and configure cert-manager integration using an
+   annotation.
+3. This kicks off the certificate issuance process – cert-manager contacts Let’s Encrypt to obtain a certificate, Let’s
+   Encrypt starts the ACME challenge, and once the domain has been verified, the certificate is issued. Cert-manager
+   stores this certificate and the signed keypair as Kubernetes resources. The keypair is stored in a Kubernetes secret
+   that is referenced by the Gateway resource.
+4. We deploy our application and our HTTPRoute which defines our routing rules. The routing rules defined configure
+   NGINX to direct requests to https://www.cafe.example.com/coffee to our coffee-app application, and to use the https
+   listener defined in our Gateway resource.
+5. NGINX is configured to terminate HTTPS traffic from clients using the signed keypair contained in the Kubernetes
```

Comment:
I think 5 is the direct result of 3. Perhaps it can be collapsed with 3 by adding there "As a result, NGINX is configured... "